src/size.rs

```diff
@@ -48,7 +56,7 @@ pub struct DisplaySize128x64;
 impl DisplaySize for DisplaySize128x64 {
     const WIDTH: u8 = 128;
     const HEIGHT: u8 = 64;
-    type BufferSize = U1024;
+    type Buffer = [u8; 1024];
```

Comment:
Any reason to not use `Self::WIDTH` and `Self::HEIGHT` here?

```suggestion
    type Buffer = [u8; Self::WIDTH as usize * Self::HEIGHT as usize / 8];
```

I'm looking forward to associated type defaults!